user_guide_src/source/concepts/services.rst

```diff
@@ -85,6 +85,8 @@ always return the same instance:
 
 .. literalinclude:: services/003.php
 
+.. note:: Since `v4.5.0`, the global function ``service()`` is recommended due to performance improvements on without parameter
```

Comment:
```suggestion
.. note:: Since v4.5.0, when you don't pass parameters to the service, the global function ``service()`` is recommended due to performance improvements.
```